### DISCUSSION TITLE
Howto "allow unsupported" in manifest mode

### DISCUSSION THREAD

**Discussion by mschollerer** [microsoft/vcpkg#35011] [Q&A] [answered]
Hi guys,

recently I switched to the manifest mode and faced with a port thats not officially supported for our plattform(s) but we know it works. In classic mode I would use the --allow-unsupported mode. But how can I handle this easily in manifest mode now?

Thanks!

**Comment by autoantwort** [comment]
Use `--allow-unsupported`

**Comment by dg0yt** [comment, marked as answer]
With CMake integration, use the `VCPKG_INSTALL_OPTIONS` cmake variable.
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration#vcpkg_install_options

With MSBuild integration, use the `VcpkgAdditionalInstallOptions` property.
https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/msbuild-integration#vcpkg-additional-install-options

**Reply by esdandreu** [reply]
Thank you for the great answer pointing the documentation. For those using CMake, at the time of writing this reply the usage of `VCPKG_INSTALL_OPTIONS` in this case would be to add `-DVCPKG_INSTALL_OPTIONS="--allow-unsupported"` to your `cmake` command.
